core/src/main/java/io/questdb/griffin/SqlCompiler.java

```diff
@@ -2777,15 +2777,17 @@ private class DatabaseBackupAgent implements Closeable {
         };
         private transient SqlExecutionContext currentExecutionContext;
         private final FindVisitor sqlDatabaseBackupOnFind = (pUtf8NameZ, type) -> {
-            if (Files.isDir(pUtf8NameZ, type, fileNameSink)) {
+            if (type == Files.DT_DIR && Files.notDots(pUtf8NameZ)) { // no backup for linked folders
```

Comment:
on second thoughts, it seems a harsh restriction not to allow backing up of partitions attached via soft link, so I am amending the check so that it matches elsewhere.